code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -844,15 +901,15 @@
 	if(isturf(H.loc)) //else, there's considered to be no light
 		var/turf/T = H.loc
 		light_amount = min(T.get_lumcount() * 10, 5)  //hardcapped so it's not abused by having a ton of flashlights
-	H.nutrition = min(H.nutrition+light_amount, NUTRITION_LEVEL_WELL_FED+10)
 
 	if(light_amount > 0)
 		H.clear_alert("nolight")
 	else
 		H.throw_alert("nolight", /obj/screen/alert/nolight)
 
-	if((light_amount >= 5) && !H.suiciding) //if there's enough light, heal
-
+	if((light_amount >= 5 && H.nutrition >= NUTRITION_LEVEL_FED ) && !H.suiciding) //if there's enough light, heal
+		if(H.nutrition >= NUTRITION_LEVEL_WELL_FED)
+			H.nutrition += min(H.nutrition+light_amount, NUTRITION_LEVEL_FED)
```

Comment:
This will exponentially increase  nutrition once it's above a certain point, giving ridiculous healing and probably breaking lots of stuff.

Reply:
Sorry, small mistake, it'll just add 350 per tick, not increase exponentially.

Reply:
well the healing is meant to be like that..the nutrition bit is what i have to try to balance out.

Reply:
open to a suggetion there